### ISSUE TITLE
utils.urlencode_noplus broken in Python 3

### ISSUE THREAD

**Issue by ze-phyr-us** [tweepy/tweepy#520]
The `'%s=%s' % ...` type of formatting in this function is not suitable for `bytes` in Python 3:

```
In [1]: import tweepy

In [2]: tweepy.utils.urlencode_noplus({'x': 'y'})
Out[2]: 'x=y'

In [3]: tweepy.utils.urlencode_noplus({'x': b'y'})
Out[3]: 'x=b%27y%27'
```

In the latter case the actual repr-ed string `b'y'` is encoded. I encountered this when using the streaming API where filter arguments are encoded to `bytes` before POSTing to statuses/filter.json.

**Comment by jason-ellis** [comment]
I've also encountered this error. Just started using tweepy today and have spent almost all day trying to figure out why I can filter "track" but not "follow". It appears to be caused by this issue. Switching to Python 2.7 interpreter allowed my module to work right away.

**Comment by jason-ellis** [comment]
Can anybody provide a workaround for this to get Stream.filter(follow) working with Python 3? I'd really like to stick with Python 3 for my app instead of resorting to Python 2.

**Comment by ze-phyr-us** [comment]
@jason-ellis You can use my fork with a Python 3-only hotfix: https://github.com/ze-phyr-us/tweepy/tree/py3-hotfix

**Comment by jason-ellis** [comment]
@ze-phyr-us Thank you so much for this! I've been banging my head against the wall for several hours about this issue.

**Comment by Harmon758** [comment]
The `utils.urlencode_noplus` method and usage of it were removed with https://github.com/tweepy/tweepy/commit/9916a548c95533758773a3ba0aa930c725ef3e39 (#558), and this change was released with Tweepy v3.3.0.
